src/gui_common/CustomRichTextLabel.cs

```diff
@@ -171,10 +180,13 @@ private void ParseCustomTags()
 
                     ThriveBbCodeTag parsedTag;
 
-                    if (Enum.TryParse(tagStack.Peek(), true, out parsedTag))
+                    if (Enum.TryParse(tagName, true, out parsedTag))
                     {
+                        // Skip the main tag
+                        var subtag = chunks.Skip(1).ToArray();
```

Comment:
I think naming wise this could be attributes to follow the convention set in html (If I understood this part correctly)

Reply:
It's indeed called attributes in some [sources](https://s9etextformatter.readthedocs.io/Plugins/BBCodes/Custom_BBCode_syntax/) I found.